open-machine-learning-jupyter-book/assignments/set-up-env/second-assignment.ipynb

```diff
@@ -26,7 +26,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
Why you submit change for this file?

Reply:
Sorry, this is the homework that was “ramped up“ before, I didn't have PR before, and I deleted him now